src/mumble/GlobalShortcut_win.h

```diff
@@ -77,6 +81,12 @@ class GlobalShortcutWin : public GlobalShortcutEngine {
 #ifdef USE_GKEY
 		GKeyLibrary *gkey;
 #endif
+#ifdef USE_XBOXINPUT
+		DWORD             xinput_last_packet[XUSER_MAX_COUNT];
+		XINPUT_STATE      xinput_state[XUSER_MAX_COUNT];
```

Comment:
Any specific reason to retain the last ```xinput_state```s here? Doesn't look like we are using them outside of the loop.